addons/slack/template/index.js

```diff
@@ -0,0 +1,58 @@
+const slack = require('./slack.js');
+const Sdk = require('@fusebit/add-on-sdk');
+const { WebClient } = require('@slack/web-api');
+const Superagent = require('superagent');
+
+module.exports = async (ctx) => {
+  const sdk = {
+    bot: new WebClient(ctx.configuration.slack_bot_access_token),
+    user: ctx.configuration.slack_user_access_token ? new WebClient(ctx.configuration.slack_user_access_token) : null,
+  };
+
+  // TODO What prevents someone who finds out this URL to
+  // send unauthorized events?
```

Comment:
Remove comment before publishing. 
Isn't the signature verification preventing unauthorized events?

Reply:
The signature check is at the connector instance that receives from Slack, not at the addon instance which receives from the connector instance. To propagate the signature checking here, I'd have to propagate the Slack signing secret to individual addon instances, which raises the story of how is it updated, which feels like going down the wrong path.

Reply:
I've filed a ticket to track

Reply:
How about we use the same bearer token used for registration?

Reply:
Good idea, fixed